src/main/java/com/github/fukkitmc/fukkit/FukkitInit.java

```diff
@@ -39,8 +39,17 @@ public void onInitialize() {
         log(Level.WARN, "Initializing");
         if(FabricLoader.getInstance().getEnvironmentType() == EnvType.CLIENT)
             LOGGER.warn("Warning, clients are not supported, loading multiple worlds at once might cause issues!");
+	    initializeMixinAddedField();
```

Comment:
Do note you have placed this in a client only block, if this can apply to dedicated servers, I would take it out of the if block

Reply:
It's already out of the block

Reply:
Must have misread some brackets then